### PR TITLE
Feat/teams bar view

### PR DESCRIPTION
**Pull request by aortiz34**
la barra de los equipos ya quedo implementada

### CODE REVIEW

File: practica-4.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <!-- Required meta tags -->
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="viewport" content="width: 30px; height: 30px;-width: 30px; height: 30px;-scale=1, shrink-to-fit=no">
```
Comment (CharlyLuna) [inline practica-4.html]: Why the change on the width and height?

**Review by CharlyLuna — CHANGES_REQUESTED** [review]
Resolve the conflicts, and please answer the comment, everything else looks good

File: practica-4.html (hunk 1 above)
Comment (aortiz34) [reply practica-4.html]: Sorry a made a change on the width and height on some lines under that ones in a group but i didn't realize that the meta tag was change.
